src/bacnet/datalink/mstp.c

```diff
@@ -1671,7 +1671,8 @@ void MSTP_Init(struct mstp_port_struct_t *mstp_port)
         if ((mstp_port->Tframe_abort < 6) || (mstp_port->Tframe_abort > 100)) {
             mstp_port->Tframe_abort = DEFAULT_Tframe_abort;
         }
-        if (mstp_port->Treply_delay > 250) {
+        if ((!mstp_port->Treply_delay) || mstp_port->Treply_delay > 250)
```

Comment:
I'd prefer to use the pattern of comparing the Treply_delay to a value (zero) instead of using the "!" operator.

Also, please fix the indents (there is a .clang-format in the root of the project that is used for the clang-format tool).